resources/views/members/form.blade.php

```diff
@@ -76,12 +79,19 @@
 		{!! Form::hidden('rijbewijs', 0) !!}
 		{!! Form::checkbox('rijbewijs', 1, null, ['style' => 'margin-top:14px;']) !!}
 	</div>
+
+	<div class="col-sm-2 form-group">
+		{!! Form::label('publish_birthday', 'Publiceer verjaardag?') !!}<br />
+		{!! Form::hidden('publish_birthday', 0) !!}
```

Comment:
Hoezo deze hidden? Ik zie dit vaker voorbij komen, maar waarom is die nodig?

Reply:
Ik weet niet of het inmiddels verouderd gedrag is, maar destijds als je een formulier instuurt met een checkbox die niet aangetikt is, dan wordt dat hele attribuut in het form request helemaal niet meegenomen. Dus dan bestaat `$request->publish_birthday` gewoon niet. Dit heb ik ooit gevonden als hack, zet een hidden default die wordt overgeschreven door een 1 als de checkbox wordt aangevinkt.

Reply:
ahh oke. Prima lekker laten staan dan, kunnen we altijd later nog opruimen